modules/monitoring-resolving-the-kubepersistentvolumefillingup-alert-firing-for-prometheus.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * monitoring/troubleshooting-monitoring-issues.adoc
+// * support/troubleshooting/investigating-monitoring-issues.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="resolving-the-kubepersistentvolumefillingup-alert-firing-for-prometheus_{context}"]
+= Resolving the KubePersistentVolumeFillingUp alert firing for Prometheus
+
+As a cluster administrator, you can resolve the `KubePersistentVolumeFillingUp` alert being triggered for Prometheus. 
+
+The critical alert fires when a persistent volume (PV) claimed by a `prometheus-k8s-*` pod in the `openshift-monitoring` project has less than 3% of its total space remaining. This can cause Prometheus to not function normally.
+
+[NOTE]
+====
+There are two `KubePersistentVolumeFillingUp` alerts:
```

Comment:
@simonpasquier I think it would be better to enhance the [Managing alerts](https://docs.openshift.com/container-platform/4.15/monitoring/managing-alerts.html) section with this information, because I definitely think it is not mentioned explicitly enough (the exact label is mentioned only in one example).

Maybe here it would be better to change it as follows?

* **Critical Alert**: The alert with `critical` severity is triggered when the mounted PV has less than 3% of its total space remaining. 
* **Warning alert**: The alert with `warning` severity  is triggered when the mounted PV has less than 15% of its total space remaining and is expected to fill up within four days.

WDYT?

Reply:
:+1: